### DISCUSSION TITLE
Change Font and size

### DISCUSSION THREAD

**Discussion by GitHubDroid** [tui-cs/Terminal.Gui#2891] [General]
Can font and size be modified?

**Comment by tznind** [comment]
Font size is a user controlled setting of the terminal itself.  Each
terminal has its own default font and size.  Typically there's a 'zoom' or
'size' setting in the terminal applications menus.

But there is no way for an app writing to the console to use different
fonts. This is simply a limitation of the protocols that govern terminals.

**Reply by GitHubDroid** [reply]
Understood.   Thank you for the great insight 👍.  We'll explained.
